android/beagle/src/main/java/br/com/zup/beagle/android/data/serializer/adapter/RouteAdapterFactory.kt

```diff
@@ -54,20 +57,25 @@ internal class RouteAdapter(private val moshi: Moshi) : JsonAdapter<Route>() {
         when (value) {
             is Route.Remote -> {
                 writer.name("url")
-                moshi.adapter(String::class.java).toJson(writer, value.url)
+                getBindAdapter().toJson(writer, value.url)
                 writer.name("shouldPrefetch")
-                moshi.adapter(Boolean::class.java).toJson(writer, value.shouldPrefetch)
+                BeagleMoshi.moshi.adapter(Boolean::class.java).toJson(writer, value.shouldPrefetch)
```

Comment:
I think you don't need to use `BeagleMoshi.moshi` you can use only `moshi`
```suggestion
moshi.adapter(Boolean::class.java).toJson(writer, value.shouldPrefetch)
```